### PR TITLE
Week1 Assignment

### PR DESCRIPTION
**Pull request by snowdenHM**
Week 1 Assignment done please review it.

### CODE REVIEW

**Comment by anilrao316** [comment]
please add @anilrao316 in description going forward so that its notified via mail

**Comment by anilrao316** [comment]
add screenshot/recording of working solution

File: Week-1/Assignment2/css/style.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+}
+
+.footer-content .footer-img-div {
+    display: flex;
+    flex: 0 0 140px;
+}
+
+.footer-content .footer-info-div {
+    display: flex;
+    flex-direction: column;
```
Comment (anilrao316) [inline Week-1/Assignment2/css/style.css:145]: lets follow BEM naming conventions https://getbem.com/naming/

File: Week-1/Assignment2/index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Catty Music Board</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" />
+    <link rel="stylesheet"
```
Comment (anilrao316) [inline Week-1/Assignment2/index.html:11]: why is this required?

Comment (snowdenHM) [reply Week-1/Assignment2/index.html:11]: In the assignment, it is mentioned that have to use the Lato font. @anilrao316
